homeassistant/components/zone/__init__.py

```diff
@@ -285,7 +295,9 @@ def __init__(self, config: dict) -> None:
         self._config = config
         self.editable = True
         self._attrs: dict | None = None
+        self._remove_listener: Callable[[], None] | None = None
         self._generate_attrs()
+        self._persons_in_zone: set[str] = set()
```

Comment:
Have we thought about counting the number of device trackers in the zone instead? Not all device trackers may be persons.

I think that zones and device trackers are the base building blocks and person is a level above. It's a bit weird to then make a base building block depend on a higher level feature.

Reply:
I honestly think its the other way around. For example, I would like to know how many people are home. E.g., if 1 person is home, things can act differently. This isn't device bound (or bound to the number of devices).

Reply:
Yeah, I agree that person is the right building block. With devices you will always be figuring out if it's the actual right devices, like iPad etc.